tests/benchmark/compute/instruction/test_system.py

```diff
@@ -462,16 +451,10 @@ def test_return_revert(
     mem_preparation = (
         Op.CODECOPY(size=return_size) if return_non_zero_data else Bytecode()
     )
-    executable_code = mem_preparation + opcode(size=return_size)
-    code = executable_code
-    if return_non_zero_data:
```

Comment:
This is lost because we are now by default returning code full of `Op.STOP`, which is all zeros. This can also be addressed by using the `code_padding_opcode` I mentioned in a previous comment (doing `code_padding_opcode=Op.INVALID` instead).